tests/test_models.py

```diff
@@ -2655,6 +2655,21 @@ def test_set_privacy(self):
         assert_false(self.project.is_public)
         assert_equal(self.project.logs[-1].action, NodeLog.MADE_PRIVATE)
 
+    @mock.patch('website.project.model.mails.queue_mail')
+    def test_set_privacy_sends_mail(self, mock_queue):
+        self.project.set_privacy('private', self.consolidate_auth)
```

Comment:
Are the first three lines of this test necessary? This test seems to be doing the same thing twice.
```self.project.set_privacy('private', self.consolidate_auth)
 self.project.set_privacy('public', self.consolidate_auth)
 assert_true(mock_queue.called_once())```